.travis.yml

```diff
@@ -2,43 +2,64 @@ language: node_js
 node_js: 
   - "8"
   - "10"
+  - "12"
 
 script: npm run unit-test
 
 jobs:
   include:
-    - stage: build
+    - &building
+      stage: build
       script: npm run build
       node_js: "8"
-    - stage: build
-      script: npm run build
+    - <<: *building
```

Comment:
This syntax isn't documented anywhere -_- but you can use this page to see what it compiles down to 
https://config.travis-ci.com/explore

Reply:
Essentially copies the stage at the `&` but you can override variables

Reply:
I think this is yaml syntax.

Reply:
Makes sense. Good to know

Reply:
We use similar syntax in our definition of the CloudFormation parameter YAML files for REVAI and RSS.